src/main/java/io/appium/java_client/android/AndroidBatteryInfo.java

```diff
@@ -0,0 +1,35 @@
+package io.appium.java_client.android;
+
+import io.appium.java_client.battery.BatteryInfo;
+
+import java.util.Map;
+
+public class AndroidBatteryInfo extends BatteryInfo {
+
+    public AndroidBatteryInfo(Map<String, Object> input) {
+        super(input);
+    }
+
+    /**
+     * @return Battery status value.
+     */
+    public BatteryStatus getStatus() {
+        final int status = ((Long) getInput().get("status")).intValue();
+        switch (status) {
+            case 2:
```

Comment:
I think it would be better to move these magic numbers to enum.

Reply:
these are used only once in this particular place. Creating a separate enum for them would be an overkill.